extensions/flac/src/main/java/com/google/android/exoplayer2/ext/flac/FlacExtractor.java

```diff
@@ -341,8 +341,7 @@ public boolean isSeekable() {
 
     @Override
     public SeekPoints getSeekPoints(long timeUs) {
-      // TODO: Access the seek table via JNI to return two seek points when appropriate.
-      return new SeekPoints(new SeekPoint(timeUs, decoderJni.getSeekPosition(timeUs)));
+      return decoderJni.getSeekPoints(timeUs);
```

Comment:
This would then return
```
SeekPoints seekPoints = decoderJni.getSeekPoints(timeUs);
return seekPoints == null ? new SeekPoints(SeekPoint.START) : seekPoints;
```